frontend/src/contexts/CanvasContext.tsx

```diff
@@ -11,13 +11,15 @@ import {
   toggleDrawingMode,
   updateDimensions,
 } from "@/utils/board/canvasUtils";
+import { ZoomOptions } from "@/enums/ZoomOptions";
+import { useZoom } from "./ZoomUIContext";
 
 const CanvasContext = createContext<ICanvasContext | undefined>(undefined);
 
 export const CanvasProvider: React.FC<{ children: React.ReactNode }> = ({ children }) => {
   const [state, dispatch] = useReducer(canvasReducer, initialState);
   const canvasRef = useRef<HTMLCanvasElement | null>(null);
-
+  const { handleZoom } = useZoom();
   useEffect(() => {
```

Comment:
I think that this `useEffect` should have the `handleZoom` function in its dependecies. At least that's what the linter is anxious about.